manifests/2/2BrightSparks/SyncBackSEX64/10.2.68.0/2BrightSparks.SyncBackSEX64.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackSEX64
+PackageVersion: 10.2.68.0
+PackageLocale: en-US
+Publisher: 2BrightSparks Pte Ltd

```

```suggestion
Publisher: 2BrightSparks
```
This must match Control Panel.